### DISCUSSION TITLE
[BUG]: Refreshing malayalam pages result in reseting configurations

### DISCUSSION THREAD

**Discussion by Ramu-kandimalla** [egovernments/Digit-Core#611] [Bugs]
https://github.com/user-attachments/assets/a717452f-b1ca-41a0-acd8-bd391a9665f0

**Comment by atulgupta2024** [comment]
the initial analysis done by the team is 

**issue in digit core, it depends on session storage (hard coded) but we had to switch to local storage (due to security issues by security audit team) and so localization doesn't load properly**

@Ramu-kandimalla please update if we have more information

**Comment by jagankumar-egov** [comment]
@atulgupta2024 Caching localizations in local storage shouldn’t be a security risk, as it only contains string translations without any personal or sensitive data, and it is cleared after session termination or logout.

 As I mentioned in previous [discussions](https://github.com/pucardotorg/dristi/issues/2280), As core digit ui components and modules being overridden, it’s unclear which specific changes have been made—whether they’re due to localization updates or new features. This remains unclear.
 

It didn't happen in our implementations

https://github.com/user-attachments/assets/eb9b5242-6925-42c2-ab25-02268863bc76

Refer to the core & library versions that we have use if it has to be reverted.
https://github.com/egovernments/DIGIT-Frontend/blob/console/health/micro-ui/web/workbench/package.json
